deploy/manifests/dev/us-east-2/cluster/kube-system/aws-auth.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: aws-auth
+data:
+  mapRoles: |
+    - groups:
+      - system:bootstrappers
+      - system:nodes
+      rolearn: arn:aws:iam::407967248065:role/default_node_group-eks-node-group-20220326104849172600000002
+      username: system:node:{{EC2PrivateDNSName}}
```

Comment:
I'm guessing this is a template variable? Where is it defined?

Reply:
This is created by EKS managed node group, i.e this specific role is managed by EKS itself along with its entry in `was-auth` config map. Here we are simply checking it into code, copied from runtime, to avoid CD overriding it.